src/Symfony/Bridge/PhpUnit/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * added log file option
```

Comment:
```suggestion
 * added log file option (defaults to `/foo/bar/name.log`) to output deprecations in a log file instead of verbose output.
```

Reply:
Thank you, I made a commit - but I left out the `defaults to /foo/bar/name.log` part because I don't understand what you mean, can you explain this?